openvdb/points/AttributeSet.cc

```diff
@@ -79,45 +75,57 @@ AttributeSet::AttributeSet(const AttributeSet& attrSet, Index arrayLength,
 
     for (const auto& namePos : mDescr->map()) {
         const size_t& pos = namePos.second;
+        const Info::Array& arrayInfo = info.arrayInfo(pos);
         Metadata::ConstPtr metadata;
         if (hasMetadata)    metadata = meta["default:" + namePos.first];
-        const AttributeArray* existingArray = attrSet.getConst(pos);
-        const bool constantStride = existingArray->hasConstantStride();
-        const Index stride = constantStride ? existingArray->stride() : existingArray->dataSize();
+        bool constantStride = arrayInfo.constantStride;
```

Comment:
Was the `const` dropped on purpose in `bool constantStride`?

Reply:
Thanks, fixed.